### PR TITLE
EXPERIMENTAL: Cleanup appdir/rootfs for pipeline

### PR DESCRIPTION
**Pull request by mikbras**

### CODE REVIEW

File: tests/Makefile [hunk 1]
```diff
@@ -132,8 +132,14 @@ dirs: $(DIRS)
...
 
+ifdef MYST_REMOVE_APPDIRS
+remove_appdir = rm -rf $(1)/appdir
+else
+remove_appdir =
+endif
+
 __tests:
-	@ $(foreach i, $(DIRS), $(MAKE) -C $(i) tests $(NL) )
+	@ $(foreach i, $(DIRS), $(MAKE) -C $(i) tests; $(call remove_appdir,$(i)) $(NL) )
```
Comment (jxyang) [inline tests/Makefile:142]: Can you add the additional step to `solutions` as well?

Comment (mikbras) [reply tests/Makefile:142]: I don't think this is going to work, which is why it is experimental. I will update soon.

**Review by radhikaj — APPROVED** [review]

File: tests/Makefile (hunk 1 above)
Comment (mikbras) [reply tests/Makefile:142]: Okay, it seems to work and I addressed your request.

**Comment by jxyang** [comment]
Nice. The pipeline shows we start from a 84%-used disk and we are still able to run through all solutions/tests without disk problem.

**Review by jxyang — APPROVED** [review]

**Pull request merged**